parser-typechecker/src/Unison/Runtime/Rt1IO.hs

```diff
@@ -489,6 +489,17 @@ toTermOrError ppe r = case r of
       P.wrap "This happens when calling a function that doesn't handle all possible inputs.",
       "", sorryMsg
       ]
+  Right (RT.RError t val) -> do
+    msg <- IR.decompile val
+    let errorType = case t of 
+                      RT.ErrorTypeTodo -> "builtin.todo" 
+                      RT.ErrorTypeBug -> "builtin.bug"
```

Comment:
Small caveat: these might not be their names forever; but we don't have a lovely way to supply these names back from the namespace to the runtime.